backend/src/activitypub/activities/handle.ts

```diff
@@ -108,12 +108,12 @@ export async function handle(
 			const objectId = getObjectAsId()
 
 			// check current object
-			const object = await objects.getObjectBy(db, 'original_object_id', objectId.toString())
```

Comment:
I think we could/should avoid these casts by making the `makeGetObjectAsId()` function return a better type.

Reply:
In fact simply removing the `Function` return type and leaving TS to infer the type resolves the problem.

Reply:
yeah totally! (sorry I wasn't digging around the code 😓 🙇 )